import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,21 @@
+import {successResponse, errorResponse} from "@libs/apiGateway";
+import {getSignedUrl} from "@libs/s3Helper";
+import {middyfy} from '@libs/lambda';
+
+
+const importProductsFile = async (event) => {
+    console.log('EVENT: ', event);
+
+    const fileName = event.queryStringParameters.name;
+    let url;
+
+    try {
+        url = await getSignedUrl(fileName);
+    } catch (e) {
+        return errorResponse(e.message)
+    }
+
+    return successResponse(url);
```

Comment:
Just an advise: you can immediately return url in `try` block right after 13 row and don't declare `url` with let 🙂